typescript/libs/popup_manager.py

```diff
@@ -263,13 +264,20 @@ def encode(str, kind):
         else:
             activeParam = ''
 
+        theme_styles = get_theme_styles(self.view)
```

Comment:
I tried this out locally and this errors

```
Traceback (most recent call last):
  File "C:\Users\Daniel\AppData\Roaming\Sublime Text 3\Packages\TypeScriptLang\typescript\libs\work_scheduler.py", line 108, in <lambda>
    sublime.set_timeout(lambda: handler(results), 0)
  File "C:\Users\Daniel\AppData\Roaming\Sublime Text 3\Packages\TypeScriptLang\typescript\libs\popup_manager.py", line 74, in <lambda>
    lambda resp: self.on_response(resp, view))
  File "C:\Users\Daniel\AppData\Roaming\Sublime Text 3\Packages\TypeScriptLang\typescript\libs\popup_manager.py", line 105, in on_response
    self.display()
  File "C:\Users\Daniel\AppData\Roaming\Sublime Text 3\Packages\TypeScriptLang\typescript\libs\popup_manager.py", line 108, in display
    popup_parts = self.get_current_signature_parts()
  File "C:\Users\Daniel\AppData\Roaming\Sublime Text 3\Packages\TypeScriptLang\typescript\libs\popup_manager.py", line 267, in get_current_signature_parts
    theme_styles = get_theme_styles(self.view)
AttributeError: 'PopupManager' object has no attribute 'view'
```

Reply:
Yep, its called different things in the two places its used..
I updated it to correctly reference `self.current_view`